### PR TITLE
License/Notice updates

### PR DESCRIPTION
**Pull request by matthiasblaesing**
This PR holds a few adjustments for license and notice files.
Please see the individual commits for further information.

If this PR is reviewed, I plan to apply the commits also to release90.

### CODE REVIEW

File: websvc.jaxws21/external/jaxws-2.2.6-notice.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Woodstox XML processor
+
+This product currently only contains code developed by authors
```
Comment (emilianbold) [inline websvc.jaxws21/external/jaxws-2.2.6-notice.txt]: I'm not sure what should we do with this notice. We are not redistributing code so this paragraph doesn't make sense.

File: websvc.jaxws21/external/jaxws-2.2.6-notice.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+of specific components, as identified by the source code files.
+
+Since product implements StAX API, it has dependencies to StAX API
```
Comment (emilianbold) [inline websvc.jaxws21/external/jaxws-2.2.6-notice.txt]: Also not sure what this paragraph is supposed to convey for downstream users.

File: websvc.jaxws21/external/jaxws-2.2.6-notice.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+classes.
+
+For additional credits (generally to people who reported problems)
```
Comment (emilianbold) [inline websvc.jaxws21/external/jaxws-2.2.6-notice.txt]: We also don't redistribute any CREDITS file that this NOTICE mentions.

**Comment by matthiasblaesing** [comment]
I agree with your assessment. I forgot, that, that file can and should be discussed. My gut feeling also was, that the file use not of much use and does not contain relevant information. In this PR I already removed the cglib NOTICE file and rethinking it I, I would also remove the jaxws notice file. The contents has indeed no value (not even referencing the product itself), so from a legal perspective I would argue, that we don't loose anything. I'll update this tonight.

**Comment by matthiasblaesing** [comment]
I pushed an updated changeset. I looked again into the jaxws zip and checked maven central. The artifacts present in the zip are also available from maven central. Having these separated, allows us to grab them from central and attribute the licenses cleaner.
This also revealed, that parts of the dependencies were in fact CDDL-1.0 (they are from the Sun days).
The three issues targetting jaxws were folded into one commit, as the changes can't be separated meaningfully.

**Comment by emilianbold** [comment]
It looks good, let me check if the files are still the same...

**Comment by emilianbold** [comment]
+1

All the hashes match (except for the javadoc ZIP but I don't mind).

It's also great we switched to Maven dependencies.

**Comment by matthiasblaesing** [comment]
For full disclosure: The javamail jar differs slightly. I ran diffoscope over the jars and the differences are:

- some of the property files contain different line endings (\n\r instead of \n)
- the file from maven central holds a LICENSE file
- the existing file holds OSGI bundle information
- the order of class files in the jar differs

the only difference from my POV are the OSGI headers, but the jaxws libraries are pulled into the classpath, so they should not matter. Will merge now.

**Pull request merged**